content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -277,7 +277,12 @@ Please note for now `db pull` will only introspect views in your schema when usi
 
 #### Unique Identifier
 
-Currently, Prisma treats views in the same way as models. This means that a view needs to have a [unique identifier](/concepts/components/prisma-schema/data-model#defining-an-id-field).
+Currently, Prisma treats views in the same way as models. This means that a view needs to have at least one _unique identifier_, which can be represented by any of the following:
+
+- An [`@id`](/concepts/components/prisma-schema/data-model#defining-an-id-field) field
+- A composite identifier denoted with [`@@id`](/concepts/components/prisma-schema/data-model#composite-ids)
```

Comment:
Let's de-emphasize those compared to `@(@)unique` somehow please. (Which is also why I wnat to get that mongoDB limitation fixed, see previous comment).

Reply:
Re-ordered that list to show the `@(@)unique` constraints first. 

Also added a sentence in the explanation beneath that list that `@(@)unique` attributes are preferable when working with a view in a SQL database and noted that MongoDB requires `@id`.